src/olympia/addons/tests/test_utils_.py

```diff
@@ -50,14 +53,51 @@ class TestGetCreaturedIds(TestCase):
     def setUp(self):
         super(TestGetCreaturedIds, self).setUp()
 
-    def test_by_category(self):
+    def test_by_category_static(self):
+        category = CATEGORIES_BY_ID[self.category]
+        assert set(get_creatured_ids(category, None)) == (
+            set(self.no_locale))
+
+    def test_by_category_dynamic(self):
```

Comment:
ugh, I didn't realise we still had non-static `Category` objects (and how are they even added?)